lib/src/chain.dart

```diff
@@ -73,7 +73,7 @@ class Chain implements StackTrace {
   ///
   /// If [callback] returns a value, it will be returned by [capture] as well.
   static T capture<T>(T Function() callback,
-      {void Function(Object error, Chain)? onError,
+      {void Function(dynamic error, Chain)? onError,
```

Comment:
Can you clarify why this should be `dynamic` now?

Reply:
This was dynamic before. I changed it in https://github.com/dart-lang/stack_trace/pull/62/files#diff-9e567343197420b24b39f009a31aa65aR76

My hope was that it wasn't breaking since any `Function(dynamic, Chain)` can work as a `Function(Object, Chain)`, but I didn't change it in the typedef here:
https://github.com/dart-lang/stack_trace/blob/90ea41ba0788c1f8c3ce1cf61a6c00f9d4d13a81/lib/src/stack_zone_specification.dart#L14

I can either change that one too, or I need to change this one back to `dynamic`.

Reply:
Given that errors can't be null any more afaik, I think changing the typedef seems reasonable? Or is there some other reason we don't want to change that?

Reply:
ok. I'll go with that